src/userANDticket/Admin.java

```diff
@@ -0,0 +1,12 @@
+package userANDticket;
+
+public class Admin extends User {
+    @Override
+    public void printRole() {
+        System.out.println("Role: userANDticket.Admin");
```

Comment:
Another variant of solving: we could create enum for role at have new field inside User